tests/acceptance/SettingsPageBasicsCest.php

```diff
@@ -0,0 +1,64 @@
+<?php
+
+ namespace MailPoet\Test\Acceptance;
+
+class SettingsPageBasicsCest {
+  function checkSettingsPagesLoad(\AcceptanceTester $I) {
+    $I->wantTo('Confirm all settings pages load correctly');
+    $I->login();
+    $I->amOnMailPoetPage('Settings');
+    //Basics Tab
+    $I->waitForText('Basics', 10);
+    $I->seeNoJSErrors();
+    //Sign-up Confirmation Tab
+    $I->click(['xpath'=>'//*[@id="mailpoet_settings_tabs"]/a[2]']);
```

Comment:
Maybe these selectors would be more self-documenting if they referred to `href` attributes like `a[@href="#signup"]`. Plus this would add a touch of robustness because we reorder these items more frequently than rename their anchors.